cfme/tests/containers/test_ssl_providers.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from itertools import product
+from cfme.containers.provider import ContainersProvider
+from cfme.common.provider import get_certificate
+from utils import testgen
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+from cfme.web_ui import fill, flash
+from cfme.fixtures import pytest_selenium as sel
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8.0.3")]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='module')
+
+DEFAULT_SEC_PROTOCOLS = 'SSL trusting custom CA', 'SSL without validation', 'SSL'
+HAWKULAR_SEC_PROTOCOLS = 'SSL trusting custom CA', 'SSL without validation', 'SSL'
+
+
+@pytest.mark.parametrize('default_sec_protocols', DEFAULT_SEC_PROTOCOLS)
+@pytest.mark.usefixtures('has_no_containers_providers')
+@pytest.mark.polarion('CMP-10586')
+def test_add_provider_ssl(provider, default_sec_protocols, soft_assert):
+    """ This test checks adding container providers with 3 different security protocols:
+    SSL trusting custom CA, SSL without validation and SSL
+        """
+    navigate_to(ContainersProvider, 'Add')
```

Comment:
please use provider object and push people to fix this issue. You don't need to reinvent the wheel because of some navigation bug.
Please create a jira card and assign it to me. I'll try to fix this issue.